python/test/t_Waarts_25_quadratic_terms.py

```diff
@@ -103,7 +103,7 @@ def cleanScalar(inScalar):
 #
 # LHS
 CoV_LHS = 0.1
-myLHS = ot.LHS(myEvent)
+myLHS = ot.ProbabilitySimulationAlgorithm(myEvent, ot.LHSExperiment())
 myLHS.setMaximumOuterSampling(8000)
```

Comment:
I suggest to increase the maximum number of function evaluations so that the CoV criteria can kick in.
```suggestion
myLHS.setMaximumOuterSampling(32000)
```